### PR TITLE
Update chrome-version

### PR DESCRIPTION
**Pull request by alexdesi**
Update Chrome version to 115

Why?
image

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -190,7 +190,7 @@ jobs:
       - checkout
       - run: sudo apt-get update
       - browser-tools/install-chrome:
```
Comment (amberstarlight) [inline .circleci/config.yml:192]:
```suggestion
      - browser-tools/install-chrome
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -190,7 +190,7 @@ jobs:
...
-          chrome-version: 114.0.5735.90 # TODO: remove -> https://github.com/CircleCI-Public/browser-tools-orb/issues/75
+          chrome-version: 124.0.6367.154 # TODO: remove -> https://github.com/CircleCI-Public/browser-tools-orb/issues/75
```
Comment (amberstarlight) [inline .circleci/config.yml:193]:
```suggestion
```

**Review by amberstarlight — COMMENTED** [review]
We could stop specifying the version here - Google keeps purging older versions from their `dl` subdomain so this will just pop up again later I think

**Comment by alexdesi** [comment]
This has been fixed.

**Pull request closed without merging**